src/components/Inventory/view/InventorySeedlingsTable.tsx

```diff
@@ -203,6 +220,45 @@ export default function InventorySeedslingsTable(props: InventorySeedslingsTable
             />
           ))}
         </Grid>
+        {selectedRows.length > 0 && (
```

Comment:
This is not necessary. You can use the showTopBar={true} on the table to do this and topBarButtons={ ... } to add the delete-batch button

Reply:
ah did not know.. will look at an example, thanks

Reply:
fixed